regression/cbmc/Float-equality1/main.c

```diff
@@ -0,0 +1,12 @@
+#include <assert.h>
+
+void main() {
+  double a, b, c;
+  __CPROVER_assume(a + b > c);
+#ifdef EQUALITY
+  double x = a, y = b, z = c;
+  assert(!(z > x + y) );
+#else 
+  assert(!(c > a + b) );
```

Comment:
This test isn't clang-format clean. (Opening `{` on new line, no space before `)` (twice).)